### PR TITLE
doc: update vUART tutorials

### PR DESCRIPTION
**Pull request by weichenli-intel**
The current code assign COM2 to S5 feature as a commutation vUART, so
the COM2's IO port which value is 0x2F8 could not assign to another
vUART connection.

This patch add a note and modify the legacy vUART sample, change the
default IO port to 0x3E8 which was known as COM3.

Tracked-On: #6690
Signed-off-by: Chenli Wei <chenli.wei@linux.intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/tutorials/vuart_configuration.rst [hunk 1]
```diff
@@ -108,7 +114,7 @@ connection between two VMs. The example extends the information provided in the
 
    #. For the vUART type, this example uses ``Legacy``.
 
-   #. For the virtual I/O address, this example uses ``0x2f8``.
+   #. For the virtual I/O address, this example uses ``0x3E8``.
```
Comment (dbkinder) [inline doc/tutorials/vuart_configuration.rst:117]: This text change says to use  ``0x3E8`` but the picture is using ``0x2E8`` and Line 143 below showing output from dmesg also is using ``0x2E8`` and talks about using ``/dev/ttyS1`` for testing the vUART communication.  These I/O addresses all need updating.

File: doc/tutorials/vuart_configuration.rst [hunk 2]
```diff
@@ -88,6 +88,12 @@ connection. Or click **-** to delete a connection.
    :align: center
    :class: drop-shadow
 
+   .. note::
```
Comment (dbkinder) [inline doc/tutorials/vuart_configuration.rst:91]: This note needs to be outdented to the left margin (that will get rid of the doc build error), and I'd suggest this wording change:

```
.. note::  The release v3.0 ACRN Configurator assigns COM2 (I/O address ``0x2F8``) to the S5 feature.
   A conflict will occur if you assign ``0x2F8`` to another connection.  In our example, we'll
   use COM3 (I/O address ``0x3F8``).


```

**Review by dbkinder — CHANGES_REQUESTED** [review]
I/O address changes to ``0x3F8`` for the example are not done everywhere in this doc.

**Review by dbkinder — APPROVED** [review]
LGTM, thanks for the update.

**Pull request merged**